packages/rust/build.sh

```diff
@@ -56,14 +66,20 @@ termux_step_configure() {
 }
 
 termux_step_make() {
-	return 0;
+	# ld: error: undefined symbol: getloadavg
+	# >>> referenced by rand.c
+        $CC $CPPFLAGS -c "$TERMUX_PKG_BUILDER_DIR"/getloadavg.c
+        $AR rcu $TERMUX_STANDALONE_TOOLCHAIN/sysroot/usr/lib/$TERMUX_HOST_PLATFORM/$TERMUX_PKG_API_LEVEL/libgetloadavg.a \
+                getloadavg.o
```

Comment:
Please place it in some rust build directory, and not in the toolchain dir. If the build is aborted, or if it fails, users would otherwise have a modified toolchain which might can cause issues